src/Employer/Employer.Web/Orchestrators/SectionsOrchestrator.cs

```diff
@@ -18,6 +20,11 @@ public SectionsOrchestrator(IVacancyClient client)
         {
             var vacancy = await _client.GetVacancyForEditAsync(vacancyId);
 
+            if (vacancy.Status == VacancyStatus.Submitted)
+            {
+                throw new ConcurrencyException($"The vacancy '{vacancy.Title}' is not available for editing.");
+            }
```

Comment:
Wondering if we should move this logic into the client otherwise we'll have to do this do this on every page.  Maybe an optional parameter on GetVacancyForEditAsync or a new function GetDraftVacancyForEditAsync()?

Reply:
if (vacancy.Status != VacancyStatus.Draft)

Reply:
There is another way around this if we were to pass a func/action but I don't think that would be favorable.

Reply:
fear not... we can move some of this stuff once we have the validation sorted